src/main/webapp/search-forms/route.tsx

```diff
@@ -38,7 +45,12 @@ const SearchForm = (props: SearchFormProps) => {
         <Dialog fullWidth maxWidth={false} open onClose={onCancel}>
           <Box height="calc(100vh - 128px)">
             <SearchFormEditor
-              form={props.form}
+              queryBuilder={queryBuilder({
```

Comment:
:pencil2: Can we use the QueryBuilder directly?
i.e.
`queryBuilder={<QueryBuilder form={form} onChange={newForm=>setForm(newForm)}/>}`

Reply:
yes :100: